requirements.txt

```diff
@@ -18,7 +18,8 @@ chardet==4.0.0
 click==7.1.2
 enum34==1.1.10
 exabyte-json-include==2.0.0
-futures==3.3.0
+futures==3.3.0;python_version<"3.0"
+futures==3.1.0;python_version>="3.0"
```

Comment:
Found this while re-making my vritualenvironment for the local docs instance. There is no build of futures 3.3.0 for Python3; 3.1.0 is the last build of futures that supported Python3.